packages/common/src/index.ts

```diff
@@ -1150,7 +1150,7 @@ export interface DbtProjectConfigBase {
 export interface DbtLocalProjectConfig extends DbtProjectConfigBase {
     type: ProjectType.DBT;
     profiles_dir?: string;
-    project_dir: string;
+    project_dir?: string;
```

Comment:
just to check my thinking: this is a safe change because we store this as a json blob and you've introduced a default prometer when generating the adapter itself (L27 above?)

Reply:
Correct